### ISSUE TITLE
Some problems with the SSID configuration.

### ISSUE THREAD

**Issue by ronaldoafonso** [cyberorg/apsetup-cockpit#1]
Hi, installed your "apsetup-cockpit" in order to do some tests and maybe use it in production but I'm facing some problems with the configuration of the SSID:

1. I cannot configure an SSID with a blank space. Like: 'Ronaldo Afonso'
2. Once one SSID is configured, I cannot change it.

  How tested is your module?

  I can help with tests and, even though, I'm not a JavaScript or Node.js programmer, I think I can help with code too.

  Thanks in advance ...

[]s
Ronaldo

**Comment by cyberorg** [comment]
yes, SSID with space is not supported, 2. is probably because of space, try without space.

I would be happy to welcome patch for space support see apsetup.js. Also AP/Adhoc modes and configuration reset that removes /etc/ap.conf and nmcli con del ap.
